buzzmobile/process/bearing/bearing.py

```diff
@@ -40,13 +49,25 @@ def bearing(fix):
         lon1 = math.radians(bearings['last_fix'].longitude)
         lat2 = math.radians(fix.latitude)
         lon2 = math.radians(fix.longitude)
-        y = math.sin(lon2 - lon1) * math.cos(lat2)
-        x = math.cos(lat1)*math.sin(lat2) - math.sin(lat1)*math.cos(lat2)*math.cos(lon2 - lon1)
-        angle = math.atan2(y, x)
-        computedBearing = (angle + 2*math.pi) % 2*math.pi
-        bearings['med_filter'].add(computedBearing)
 
-        bearing_pub.publish(bearings['med_filter'].median())
+        #Calculates great-circle distance between positions
+        a = (math.pow(math.sin(lat2 - lat1), 2)
```

Comment:
Can you add units to the comment? Is a in meters? radians? km?

Also maybe factor out this great-distance calculation.